traitsui/editors/video_editor.py

```diff
@@ -39,50 +40,46 @@ class VideoEditor(BasicEditorFactory):
     #: doesn't match the aspect ratio of the video stream.
     aspect_ratio = AspectRatio()
 
-    #: The name of a trait to synchronise with the audio muted state of
-    #: the video.  The referenced trait should be a Bool.
-    muted = Str(sync_value='from', sync_name='muted')
+    #: True if the audio is muted, False otherwise
+    muted = CVType(Bool(), default=False, sync_value='from')
```

Comment:
Style nit - I think that for inner traits we prefer no parens unless needed, so `CVType(Bool, default..., etc.)`.  I think the typing stubs handle it better that way.

Reply:
Fixed